fil_actors_shared/src/v9/util/mod.rs

```diff
@@ -16,3 +19,79 @@ mod mapmap;
 mod multimap;
 mod set;
 mod set_multimap;
+
+/// Converts address key to key, equivilent go code can be found at L54-L61 in
```

Comment:
I need more than this documentation for me to understand. What kind of key do we convert to? How is `key` different from `address key`? What even is an address key? Why doesn't this function take an `Address` instead of a `BytesKey`?

Reply:
Unfortunately, I'm not able to find code comments or docs that explain why they're converting the hamt key like this.  Basically the code loads a hamt map, and converts the key with this function and new key for a few internal hamt maps

Reply:
I'm OK with not merging this and keep it in `forest` repo, just wanted to create parity test to make sure my implementation is correct